### ISSUE TITLE
Make `TabIndex` an DesignableProperty

### ISSUE THREAD

**Issue by tznind** [gui-cs/TerminalGuiDesigner#79]
> I think the view position do not guarantee a tab order because it's handled by the `View.TabIndex` property. The user may want to change another tab order than the position. Should be a good idea to give to the user a way to change the `TabIndex` on the view properties UI.

_Originally posted by @BDisp in https://github.com/gui-cs/TerminalGuiDesigner/issues/78#issuecomment-1232776365_

**Comment by BDisp** [comment]
I also think a good idea to add the same to the `TabStop`.
